python_ta/checkers/redundant_assignment_checker.py

```diff
@@ -34,33 +34,47 @@ class RedundantAssignmentChecker(BaseChecker):
     name = "redundant_assignment"
     msgs = {
         "E9959": (
-            "This assignment statement is redundant;" " You can remove it from the program.",
+            "This assignment statement for variable(s) `%s` is redundant;"
```

Comment:
Reword to: "Assigning to variable(s) %s here is redundant, because they are not used before getting reassigned later. You can remove the assignment(s) without changing the behaviour of this code."

Please also update the "help" message text on Line 40 below.